src/test/scala/wdlTools/eval/EvalTest.scala

```diff
@@ -259,34 +260,47 @@ class EvalTest extends AnyFlatSpec with Matchers with Inside {
 
   it should "evaluate simple command section" in {
     val command = evalCommand("command_simple.wdl")
-    command.trim shouldBe "We just discovered a new flower with 100 basepairs. Is that possible?"
+    command shouldBe "We just discovered a new flower with 100 basepairs. Is that possible?"
   }
 
   it should "evaluate command section with some variables" in {
     val command = evalCommand("command2.wdl")
-    command.trim shouldBe "His trumpet playing is not bad"
+    command shouldBe "His trumpet playing is not bad"
   }
 
   it should "command section with several kinds of primitives" in {
     val command = evalCommand("command3.wdl")
-    command.trim shouldBe "His trumpet playing is good. There are 10 instruments in the band. It this true?"
+    command shouldBe "His trumpet playing is good. There are 10 instruments in the band. It this true?"
   }
 
   it should "separator placeholder" in {
     val command = evalCommand("sep_placeholder.wdl")
-    command.trim shouldBe "We have lots of numbers here 1, 10, 100"
+    command shouldBe "We have lots of numbers here 1, 10, 100"
   }
 
   it should "boolean placeholder" taggedAs Edge in {
     val command = evalCommand("bool_placeholder.wdl")
-    val lines = command.split("\n")
-    lines(1).trim shouldBe "--no"
-    lines(2).trim shouldBe "--yes"
+    command shouldBe s"--no${linesep}--yes"
   }
 
   it should "default placeholder" taggedAs Edge in {
     val command = evalCommand("default_placeholder.wdl")
-    command.trim shouldBe "hello"
+    command shouldBe "hello"
+  }
+
+  it should "strip common indent" taggedAs Edge in {
+    val command = evalCommand("indented_command.wdl")
+    command shouldBe " echo 'hello Steve'\necho 'how are you, Steve?'\n   echo 'goodbye Steve'"
+  }
+
+  it should "strip common indent in python heredoc" taggedAs Edge in {
```

Comment:
Great, this is the kind of use I am worried about. Now we have a unit test for it.